besu/src/main/java/org/hyperledger/besu/cli/options/TransactionPoolOptions.java

```diff
@@ -195,6 +197,39 @@ static class Legacy {
     Integer txPoolMaxSize = TransactionPoolConfiguration.DEFAULT_MAX_PENDING_TRANSACTIONS;
   }
 
+  @CommandLine.ArgGroup(validate = false)
+  private final TransactionPoolOptions.Unstable unstableOptions =
+      new TransactionPoolOptions.Unstable();
+
+  static class Unstable {
+    private static final String TX_MESSAGE_KEEP_ALIVE_SEC_FLAG =
+        "--Xincoming-tx-messages-keep-alive-seconds";
+
+    private static final String ETH65_TX_ANNOUNCED_BUFFERING_PERIOD_FLAG =
+        "--Xeth65-tx-announced-buffering-period-milliseconds";
```

Comment:
Again this seems to be unrelated to this PR?

Reply:
I put this here since, I am in the process of refactoring how we manage the option, and so when I took some, I also take the opportunity to iterate on it, in this case it was the merging of (un)stable options in a single place.